packages/mdl-ripple/mixin.js

```diff
@@ -0,0 +1,313 @@
+import {SequentialAnimation} from './animation';
+
+
+/** @typedef {string} */
+var ElementRef;
```

Comment:
I feel like we should probably keep this closure-specific code out of our source code for now, as it'll probably confuse those unfamiliar 😃